source/install-atlas-cli.txt

```diff
@@ -210,77 +210,126 @@ Follow These Steps
                .. tab:: MongoDB Community Edition
                   :tabid: mdb-comm
 
-                  Create the list file
-                  ``/etc/apt/sources.list.d/mongodb-org-5.0.list`` for
-                  your version of Ubuntu. Replace ``5.0`` with your
-                  edition of MongoDB.
+                  Select Ubuntu or Debian, then select your version.
 
                   .. tabs::
 
-                     .. tab:: Ubuntu 20.04 (Focal)
-                        :tabid: comm-focal
+                     .. tab:: Ubuntu
+                        :tabid: ubuntu
+
+                        Create the list file 
+                        ``/etc/apt/sources.list.d/mongodb-org-
+                        {+mdbVersion+}.list`` for your version of
+                        Ubuntu. Replace ``{+mdbVersion+}`` with your
+                        edition of MongoDB.
+
+                        .. tabs::
+
+                           .. tab:: Ubuntu 22.04 (Jammy)
+                              :tabid: comm-jammy
+
+                              .. code-block:: sh
+
+                                 echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu jammy/mongodb-org/{+mdbVersion+} multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-{+mdbVersion+}.list
+
+                           .. tab:: Ubuntu 20.04 (Focal)
+                              :tabid: comm-focal
+
+                              .. code-block:: sh
+
+                                 echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/{+mdbVersion+} multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-{+mdbVersion+}.list
+
+                           .. tab:: Ubuntu 18.04 (Bionic)
+                              :tabid: comm-bionic
 
-                        .. code-block:: sh
+                              .. code-block:: sh
 
-                           echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/5.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-5.0.list
+                                 echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/{+mdbVersion+} multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-{+mdbVersion+}.list
 
-                     .. tab:: Ubuntu 18.04 (Bionic)
-                        :tabid: comm-bionic
+                     .. tab:: Debian
+                        :tabid: debian
 
-                        .. code-block:: sh
+                        Create the list file 
+                        ``/etc/apt/sources.list.d/mongodb-org-
+                        {+mdbVersion+}.list`` for your version of
```

Comment:
**Change**: This renders in the staging version with an extra space between `org-` and `6.0.list`.

Reply:
Fixed, thanks!